draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -1540,19 +1540,6 @@ https://www.example.org
   TE:
   TE: trailers, deflate;q=0.5
 </artwork>
-<t>
-   The presence of the keyword "trailers" indicates that the client is willing
-   to accept trailer fields in a chunked transfer coding, as defined in
-   <xref target="chunked.trailer.section"/>, on behalf of itself and any downstream
-   clients. For requests from an intermediary, this implies that either:
-   (a) all downstream clients are willing to accept trailer fields in the
-   forwarded response; or,
-   (b) the intermediary will attempt to buffer the response on behalf of
-   downstream recipients.
-   Note that HTTP/1.1 does not define any means to limit the size of a
-   chunked response such that an intermediary can be assured of buffering the
-   entire response.
-</t>
```

Comment:
We seem to lose the note in the last paragraph - is this acceptable?

Reply:
Personally I think it's OK to lose it; it's a logical conclusion of the design that someone can infer on their own.

Reply:
Ok. Maybe move that statement to "Chunked Encoding" instead?